tensorflow_probability/python/experimental/bijectors/__init__.py

```diff
@@ -17,9 +17,13 @@
 from tensorflow_probability.python.bijectors.ldj_ratio import inverse_log_det_jacobian_ratio
 from tensorflow_probability.python.experimental.bijectors.distribution_bijectors import make_distribution_bijector
 from tensorflow_probability.python.experimental.bijectors.scalar_function_with_inferred_inverse import ScalarFunctionWithInferredInverse
+from tensorflow_probability.python.experimental.bijectors.highway_flow import build_highway_flow_layer, HighwayFlow
```

Comment:
Google's Python style prefers only one import per line, even if that means redundant lines. Here, you'll need to write:

```
from tensorflow_probability.python.experimental.bijectors.highway_flow import build_highway_flow_layer 
from tensorflow_probability.python.experimental.bijectors.highway_flow import HighwayFlow
```